core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
https://libgdx.badlogicgames.com/nightlies/docs/api/com/badlogic/gdx/Application.html

It makes it thread safe by delegating the work to the engine, which manages the threads. If we create our own logic without letting the engine control when it happens we can have memory leaks and runaway programs because our code will be independent of the framework. We don't need it for every class we create because they don't control game logic but the TriviaButtonBuilder class does. Especially because it has an InputListener.